04_numeronym/srini/main_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var testMatrix = []struct {
+	input  []string
+	output []string
+}{
+	{[]string{"abba", "accessibility", "Kubernetes", "abc"}, []string{"a2a", "a11y", "K8s", "abc"}},
```

Comment:
Please add some unicode character tests.

Reply:
Updated to support unicode

Reply:
Fixed code to support unicode and added test corresponding test cases